modes/pdf/evil-collection-pdf.el

```diff
@@ -195,6 +217,13 @@ Consider COUNT."
   (evil-collection-define-key 'visual 'pdf-view-mode-map
     "y" 'pdf-view-kill-ring-save)
 
+  (evil-collection-inhibit-insert-state 'pdf-history-minor-mode-map)
+  (evil-set-initial-state 'pdf-history-minor-mode 'normal)
+  (evil-collection-define-key 'normal 'pdf-history-minor-mode-map
+    ;; history forward / backward
+    (kbd "C-i") 'evil-collection-pdf-jump-forward
+    (kbd "C-o") 'evil-collection-pdf-jump-backward)
```

Comment:
Does it work? Most minors need `evil-normalize-keymaps`. e.g. https://github.com/emacs-evil/evil-collection/blob/master/modes/auto-package-update/evil-collection-auto-package-update.el#L38

Reply:
It turned out that it really didn't stop working after a clean run. For some reason,
the original version doesn't have these hooks for the other minor modes either
(`pdf-outline-buffer-mode` and `pdf-occur-buffer-mode-map`).
But `pdf-outline-buffer-mode` bindings works...

I can add `evil-normalize-keymaps` if needed, as far as I understand, this should
be done for each minor mode separately.

Reply:
Sorry for misleading, I confirm that this works the bindings work without
`evil-normalize-keymaps` for `pdf-history-minor-mode`. What confuses me
is that `pdf-tools`'s jump history are page-accurate, not character-accurate
like in Vim. You can see this in the header using the `pdf-history-debug` function.

Reply:
OK.